src/components/site/Breadcrumbs.md

```diff
@@ -0,0 +1,84 @@
+### Breadcrumbs Light
+
+```jsx
+import Breadcrumbs from "./Breadcrumbs";
+import BreadcrumbItem from "./Breadcrumbitem";
+import BreadcrumbLinkItem from "./BreadcrumbLinkitem";
+<div class="dg_section">
+  <div class="container">
+    <Breadcrumbs>
+      <BreadcrumbLinkItem
+        breadcrumbText="Departments"
+        title="This is a title"
+        link="#"
+      />
+      <BreadcrumbLinkItem
+        breadcrumbText="Department of Health"
+        title="This is a title"
+        link="#"
+      />
+      <BreadcrumbItem breadcrumbText="Adoptable Pets" />
+    </Breadcrumbs>
+  </div>
+</div>;
+```
+
+Html Snippet:
+
+```html
+<div class="dg_section">
+  <div class="container">
+    <ul class="dg_breadcrumbs">
+      <li>
+        <a href="#"><i class="fas fa-home fa-1x" aria-hidden="true"></i></a>
+      </li>
+      <li><a title="This is a title" href="#">Departments</a></li>
+      <li><a title="This is a title" href="#">Department of Health</a></li>
+      <li>Adoptable Pets</li>
+    </ul>
+  </div>
+</div>
+```
+
+### Breadcrumbs Dark
+
+```jsx
+import Breadcrumbs from "./Breadcrumbs";
```

Comment:
not blocking but you should need this import

Reply:
should not*

Reply:
Updated and committed. The import for /Breadcrumbs has been removed from the .md file.